### DISCUSSION TITLE
Dynamic Routing for Shared Common Directory Content

### DISCUSSION THREAD

**Discussion by HeilOliver** [vuejs/vitepress#4441] [Q&A]
I have a project structure like this
```
en/
  |- common/
      |- commonA.md
      |- commonB.md
  |- partA/
  |- partB/
```

What i want is at the end routes for 
```
en/partA/common/commonA
en/partA/common/commonB
en/partB/common/commonA
en/partB/common/commonB
```

How can I archive this?

I have already tried the dynamic routes, but creating a js file for each md file is quite ugly. 
Can I do this at folder level? Or is there one paths.js that does apply to all?

**Comment by brc-dd** [comment]
Yes you can have single paths.js file do it - https://stackblitz.com/edit/vite-e8qc54ru?file=docs/%5Bid%5D.paths.ts
